engine/src/main/java/org/hibernate/search/engine/search/predicate/DslConverter.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.search.predicate;
+
+public enum DslConverter {
```

Comment:
If this class is not used in any API, I'd rather move it to an SPI package for now. Let's not expose anything to users unnecessarily, that will just prevent us from changing it...

Reply:
Good idea!